src/main/java/org/example/homework3/Calculator.java

```diff
@@ -0,0 +1,27 @@
+package org.example.homework3;
+public class Calculator {
+
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 9;
+
+        //Sum
```

Comment:
I guess those comments are not needed. The naming of variables is pretty clear, what is the purpose of that comment? You can leave it as it is, no problem. But comments are needed only if the code is not obvious and straightforward

Reply:
Removed